sdk/python/kfp_tekton/compiler/yaml_utils.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Union, Optional, TextIO, overload, OrderedDict
```

Comment:
do you know when did `OrderedDict` introduced in Python. It was failing on python 3.5, 3.6 and 3.7.

we will be deprecating 3.5 soon, but kfp wants to support python 3.6+.

Reply:
can we workaround with the collections library?

Reply:
We sure can. Frankly speaking, I didn't even notice it was imported from `typing` and not `collection` here.